server/src/testFixtures/java/io/crate/testing/SQLTransportExecutor.java

```diff
@@ -468,6 +474,21 @@ private static SQLResponse executeAndConvertResult(PreparedStatement preparedSta
         }
     }
 
+    /**
+     * Map type name from jdbc response (metadata) to a DataType
+     * This roughly follows {@link PGTypes}
+     */
+    private static DataType<?> getDataType(String pgTypeName) {
+        DataType<?> dataType = switch (pgTypeName) {
```

Comment:
have you considered making this method complete for all types?

Reply:
yes I had. Given that it's only test logic I think it's fine to extend as needed.
We hardly ever assert the column types in integration tests. That's usually covered via analyzer or planner tests.